r2/r2/models/link.py

```diff
@@ -408,9 +408,6 @@ def slug():
             res = "/%s" % p
         return res
 
-    def make_permalink_slow(self):
-        return self.make_permalink(self.subreddit_slow)
-
```

Comment:
For the record, I stared blankly at the screen for a while before that occurred to me, and I've been writing software professionally over 20 years, so don't feel bad.